### PR TITLE
statistics: migrate test-infra to testify for analyze_jobs_test.go

### LINKED ISSUE #28116: migrate test-infra to testify for `statistics/analyze_jobs_test.go`

### PR DESCRIPTION
**Pull request by feitian124**
### What problem does this PR solve?

Issue Number: close #28116 

### Release note

```release-note
None
```

### CODE REVIEW

**Review by tisonkun — COMMENTED** [review]
We can rename `analyze_jobs_test.go` to `analyze_jobs_serial_test.go`. Rest LGTM.

**Comment by feitian124** [comment]
> We can rename `analyze_jobs_test.go` to `analyze_jobs_serial_test.go`. Rest LGTM.

sorry, this file only one test i forgot add `t.Parallel()`.

updated, please review again.

**Review by mmyj — APPROVED** [review]
lgtm

**Comment by mmyj** [comment]
PTAL @tisonkun

File: statistics/analyze_jobs_test.go [hunk 1]
```diff
@@ -15,14 +15,13 @@
...
+	"testing"
 
-type testStatisticsSerialSuite struct{}
+	"github.com/stretchr/testify/require"
+)
 
-func (s *testStatisticsSerialSuite) TestMoveToHistory(c *C) {
+func TestMoveToHistory(t *testing.T) {
+	t.Parallel()
 	ClearHistoryJobs()
```
Comment (tisonkun) [inline statistics/analyze_jobs_test.go:25]: @feitian124 This method seems touch global state and the original test run in serial. Could you elaborate the reason change the behavior?

Comment (feitian124) [reply statistics/analyze_jobs_test.go]: sorry, my fault, should run in serial.
updated.

**Review by tisonkun — APPROVED** [review]

**Pull request merged**